tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Services/EmailAddressTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using FluentAssertions;
+using MimeKit;
+using NHSD.BuyingCatalogue.Organisations.Api.Services;
+using NUnit.Framework;
+
+namespace NHSD.BuyingCatalogue.Organisations.Api.UnitTests.Services
+{
+    [TestFixture]
+    [Parallelizable(ParallelScope.All)]
+    internal sealed class EmailAddressTests
+    {
+        [Test]
+        public void Constructor_String_String_InitializesExpectedValues()
```

Comment:
I think this might be a typo.

Reply:
What's that – `String_String`? That's actually deliberate – it corresponds to the signature of the constructor.

Reply:
😅 Ignore what I said

Reply:
Good to check though!